### PR TITLE
Fix errors from 'ansible-test sanity' checking

### PR DESCRIPTION
**Pull request by hsong-rh**
This PR fix the errors reported when running `ansible-test sanity` under the ansible collection location.

https://issues.redhat.com/browse/AAP-5047

### CODE REVIEW

**Comment by Alex-Izquierdo** [comment]
It's still returning two errors:
```sh
ERROR: Found 1 shebang issue(s) which need to be resolved:
ERROR: eda/plugins/modules/upcase.py:1:1: unexpected non-module shebang: b'#!/usr/bin/python'
See documentation for help: https://docs.ansible.com/ansible-core/2.12/dev_guide/testing/sanity/shebang.html
ERROR: Found 1 yamllint issue(s) which need to be resolved:
ERROR: eda/galaxy.yml:62:1: empty-lines: too many blank lines (1 > 0)
See documentation for help: https://docs.ansible.com/ansible-core/2.12/dev_guide/testing/sanity/yamllint.html
ERROR: The 2 sanity test(s) listed below (out of 46) failed. See error output above for details.
shebang
yamllint
```

File: benthomasson/eda/plugins/modules/upcase.py [hunk 1]
```diff
@@ -1,17 +1,32 @@
 #!/usr/bin/python
 
+# This file is part of Ansible
```
Comment (benthomasson) [inline benthomasson/eda/plugins/modules/upcase.py:3]: Is this true?  It isn't part of Ansible core.  Do we apply GPL to all our files?

Comment (hsong-rh) [reply benthomasson/eda/plugins/modules/upcase.py:3]: This is the only file complained missing license information. I copied it from other files, may need to be updated for our purpose

Comment (benthomasson) [reply benthomasson/eda/plugins/modules/upcase.py:3]: Please change the license to Apache v2

**Review by benthomasson — CHANGES_REQUESTED** [review]

**Comment by hsong-rh** [comment]
@benthomasson To pass the validation, it needs GPLv3 license
```
Running sanity test "validate-modules"
ERROR: Found 1 validate-modules issue(s) which need to be resolved:
ERROR: plugins/modules/upcase.py:0:0: missing-gplv3-license: GPLv3 license header not found in the first 20 lines of the module
See documentation for help: https://docs.ansible.com/ansible-core/2.13/dev_guide/testing/sanity/validate-modules.html
```

**Review by Alex-Izquierdo — APPROVED** [review]

File: benthomasson/eda/plugins/modules/upcase.py (hunk 1 above)
Comment (mkanoor) [reply benthomasson/eda/plugins/modules/upcase.py:3]: @hsong-rh @benthomasson That is a test module I created. So that we can test it

Comment (hsong-rh) [reply benthomasson/eda/plugins/modules/upcase.py]: @mkanoor The line `# This file is part of Ansible` is removed

**Pull request merged**